src/shared/views/BuildingFootprint/FootprintDisplay.js

```diff
@@ -112,23 +222,88 @@ export default class FootprintDisplay extends React.Component<Props> {
+                    onLayout={this.onLayout}
                     style={{
-                        height: tileSize,
-                        width: tileSize,
+                        flex: 1,
+                        // height: 1,
+                        width: GLOBAL.SCREEN_WIDTH,
                     }}
                 />
             );
         }
         const coords = task.geojson.coordinates[0];
-        const center = this.getBuildingCentroid(coords);
+
+        if (project.tileServer.url.includes('googleapis')) {
+            // google imagery is returned as a single image of the size we want
+            // so we need a different logic, as we can't just pull 4 images
+            // (each call costs money, and would include a credit line)
+            const googleSize = `${this.imagerySize}x${this.imagerySize}`;
+            // some projects include a `center` attribute in the task which defines
+            // the center point of the imagery to use. This allows some optimisation
+            // of number of imagery requests by reusing the same image for multiple
+            // tasks.
+            let googleCenter; // the center of the imagery
+            let center; // the geometry center
+            if (task.center) {
+                googleCenter = `${task.center[1]}%2C%20${task.center[0]}`;
+                center = task.center;
+            } else {
+                center = this.getTaskGeometryCentroid(coords);
+                googleCenter = `${center[1]}%2C%20${center[0]}`;
+            }
+            const p = this.getGooglePolygonFromCenter(
+                center,
+                zoomLevel,
+                coords,
+            );
+            // tileUrl -> imageryUrl
+            const tileUrl = project.tileServer.url
+                .replace('{z}', zoomLevel.toString())
+                .replace('{size}', googleSize)
+                .replace('{center}', googleCenter);
+            return (
+                <View
+                    {...this.panResponder.panHandlers}
+                    style={{
+                        alignSelf: 'center',
+                        flex: 1,
+                        overflow: 'hidden',
+                        aspectRatio: 1,
+                    }}
+                >
+                    <Image
+                        style={[
+                            {
+                                left: 0,
+                                height: this.imagerySize,
+                                width: this.imagerySize,
+                                top: 0,
+                            },
+                            styles.tileImg,
+                        ]}
+                        source={{ uri: tileUrl }}
+                    />
+                    <Surface
+                        height={GLOBAL.SCREEN_WIDTH}
+                        width={GLOBAL.SCREEN_WIDTH}
+                    >
+                        <Shape d={p} stroke="red" strokeWidth={1} />
```

Comment:
@jhenshall red or any other color?

Reply:
This is probably a question to check with regards to accessibility. I guess an option could be 3 lines white/black/white to have a very strong contrast. See #252